### PR TITLE
Docs: Modular Movement section

### PR DESCRIPTION
**Pull request by camiinthisthang**

### CODE REVIEW

File: src/modular-movement.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+There is an ongoing colossal shift happening in layer-1 (L1) blockchain architectures. We are moving away from a monolithic design, where consensus, data availability, and execution are tightly coupled (e.g. today's Ethereum), to a modular future, where execution is separated from data availability and consensus (e.g. tomorrow's Eth2, or Celestia). This separation allows for specialization at the base layer, delivering a significant increase in bandwidth capacity.
+
+
+## Why Modular?
+
+A modular blockchain architecture does not inherently enable scaling. The properties that are derived as a result are what make this possible. Fuel was built for fraud proofs, enabling trust-minimized light clients, enabling high security without requiring high resource usage.
+
+### Security vs. Resource Requirements
+
+![fuel1](https://user-images.githubusercontent.com/15346823/187789550-382eb91d-1a1d-4cf7-a26c-c82e6e8a12f9.png)
```
Comment (adlerjohn) [inline src/modular-movement.md]: Ditto on local images

**Review by adlerjohn — COMMENTED** [review]
Ditto on linting fixes in this file.

File: src/modular-movement.md (hunk 1 above)
Comment (camiinthisthang) [reply src/modular-movement.md]: done

**Comment by camiinthisthang** [comment]
linter failing but in files outside of this PR, will fix those in the appropriate branches

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**